dev/import-beats/kibana.go

```diff
@@ -19,12 +19,22 @@ import (
 )
 
 var (
-	encodedFields = []string{
+	fieldsToEncode = []string{
+		"attributes.kibanaSavedObjectMeta.searchSourceJSON",
+		"attributes.optionsJSON",
+		"attributes.panelsJSON",
 		"attributes.uiStateJSON",
 		"attributes.visState",
+	}
+
+	fieldsToDecode = []string{
+		"attributes.kibanaSavedObjectMeta.searchSourceJSON",
+		"attributes.layerListJSON",
+		"attributes.mapStateJSON",
 		"attributes.optionsJSON",
 		"attributes.panelsJSON",
-		"attributes.kibanaSavedObjectMeta.searchSourceJSON",
+		"attributes.uiStateJSON",
```

Comment:
Why is encoded and decode not the same list?

Could you import this list from here https://github.com/elastic/package-registry/blob/master/dev/generator/main.go#L253 so we have a single point of truth?

Reply:
It's not the same list, because some fields are already encoded in dashboards stored in Beats (see mapStateJSON in https://github.com/elastic/beats/blob/master/x-pack/filebeat/module/o365/_meta/kibana/7/dashboard/Filebeat-O365-Audit.json#L842).

Another solution would be detecting if the field is "string" and then try to decode it. WDYT?

Reply:
Oh, I see. So we basically missed it already in Beats, I wonder if we should add it there @exekias .

A check for string would be nice as it would mean we only have to keep one list and things to break if the dashboards in Beats itself are adjusted.

Reply:
Fixed!